main.go

```
@@ -1,7 +1,149 @@
+	log.Println("Server is running on port " + cfg.port)
+	log.Fatal(srv.ListenAndServe())
 }
+
+func renderTemplate(w http.ResponseWriter, tmpl string, data interface{}) {
+	fmt.Println("templates/" + tmpl)
+	t, err := template.ParseFiles("templates/" + tmpl)
+	if err != nil {
+		http.Error(w, "Template error", http.StatusInternalServerError)
+		return
+	}
+	t.Execute(w, data)
```

golangci-lint (1.62.2)

Error return value of `t.Execute` is not checked

(errcheck)